srcpkgs/chatterino2/patches/0001-use-system-deps.patch

```diff
@@ -1,58 +0,0 @@
-diff --git chatterino.pro chatterino.pro
-index d8b14bbe..f7657f06 100644
---- a/chatterino.pro
-+++ b/chatterino.pro
-@@ -12,10 +12,17 @@
- #  - BOOST_DIRECTORY (C:\local\boost\ by default) (Windows only)
-
- QT                += widgets core gui network multimedia svg concurrent
--CONFIG            += communi
--COMMUNI           += core model util
-
- INCLUDEPATH       += src/
-+# libcommuni
-+INCLUDEPATH       += /usr/include/qt5/Communi/IrcCore
-+INCLUDEPATH       += /usr/include/qt5/Communi/IrcModel
-+INCLUDEPATH       += /usr/include/qt5/Communi/IrcUtil
-+DEFINES           += IRC_SHARED
-+LIBS              += -lIrcCore -lIrcModel -lIrcUtil
-+# qtkeychain
-+INCLUDEPATH       += /usr/include/qt5keychain
-+LIBS              += -lqt5keychain
- TARGET             = chatterino
- TEMPLATE           = app
- PRECOMPILED_HEADER = src/PrecompiledHeader.hpp
-@@ -78,7 +85,6 @@ CONFIG(debug, debug|release) {
- include(lib/warnings.pri)
- include(lib/fmt.pri)
- include(lib/humanize.pri)
--include(lib/libcommuni.pri)
- include(lib/websocketpp.pri)
- include(lib/wintoast.pri)
- include(lib/signals.pri)
-@@ -86,7 +92,6 @@ include(lib/settings.pri)
- include(lib/serialize.pri)
- include(lib/winsdk.pri)
- include(lib/rapidjson.pri)
--include(lib/qtkeychain.pri)
```

Comment:
I see now from this patch that at least libcommuni and qt keychain didn't need to be submodules. Did you try rebasing the patch?

Reply:
The patch is for qmake, Chatterino switched to cmake since then, libcommuni doesn't support cmake and Chatterino's fork is used. Not sure about qtkeychain

Reply:
qtkeychain is also a fork, not sure if it will compile with upstream version https://github.com/Chatterino/qtkeychain/commits/de954627363b0b4bff9a2616f1a409b7e14d5df9

Reply:
Upstream version might not compile because of our custom hacks in Chatterino's cmake implementation. Chatterino explicitly uses Chatterino/qtkeychain and this is what should be used instead of upstream qtkeychain version.